packages/core/src/sdk.ts

```diff
@@ -48,15 +47,15 @@ export function setCurrentClient(client: Client): void {
 }
 
 /**
- * Unfortunately, we still have to manually bind the client to the "hub" property set on the global
+ * Unfortunately, we still have to manually bind the client to the "stack" property set on the global
  * Sentry carrier object. This is because certain scripts (e.g. our loader script) obtain
- * the client via `window.__SENTRY__.hub.getClient()`.
+ * the client via `window.__SENTRY__[version].stack.getClient()`.
  *
  * @see {@link ./asyncContext/stackStrategy.ts getAsyncContextStack}
  */
 function registerClientOnGlobalHub(client: Client): void {
-  const sentryGlobal = getSentryCarrier(getMainCarrier()) as { hub?: AsyncContextStack };
-  if (sentryGlobal.hub && typeof sentryGlobal.hub.getStackTop === 'function') {
-    sentryGlobal.hub.getStackTop().client = client;
+  const sentryGlobal = getSentryCarrier(getMainCarrier());
+  if (sentryGlobal.stack && typeof sentryGlobal.stack.getStackTop === 'function') {
+    sentryGlobal.stack.getStackTop().client = client;
   }
```

Comment:
maybe safes some bytes if we just do this:

```suggestion
  try {
    sentryGlobal.stack.getStackTop().client = client;
 } catch {
   // ignore errors here
 }
```

as this _should_ always be there, and else we can just try-catch all of this...?